File: python/FizzBuzz.py
# The 'seminal' FizzBuzz problem
# from -50 to 100...in python. 
# Almost too easy...it works!
printed = 0
i = -50
while i <=100: 
    if (i % 3 == 0) and (i % 5 ==0): 
        print("FizzBuzz")
        printed = 1
    if (i % 3 == 0) and (printed <> 1):
        print("Fizz")
    if (i % 5 == 0) and (printed <> 1):
        print("Buzz")
    else:
        print (i)
    i += 1
# eof

File: Java/fizzbuzz.java
/*   A simple FizzBuzz solution in Java
     if number is / 3 print Fizz
     if number is / 5 print Buzz
     if number is / 3 and 5 both print FizzBuzz
     else print the number
     This one is better than my somewhat poor python solution
     Less repitition here...still not a great piece of code but it works
*/
    
public class FizzBuzz{

     public static void main(String []args){
         int low = -50, high = 50;
         for (int i = low; i <= high; i++) {
             String test = "";
             test += (i % 3) == 0 ? "Fizz" : "";
             test += (i % 5) == 0 ? "Buzz" : "";
             System.out.println(!test.isEmpty() ? test: i);
         }
     }
}
